code/game/machinery/vending.dm

```diff
@@ -1504,12 +1511,8 @@
 	to_chat(user, SPAN_WARNING("[src] flashes a message: Unauthorized Access."))
 	return FALSE
 
-/obj/machinery/vending/theomat/vend(datum/data/vending_product/R, mob/user)
-	if(check_NT(user))
-		..()
-
-/obj/machinery/vending/theomat/try_to_buy(obj/item/W, var/datum/data/vending_product/R, var/mob/user)
-	if(check_NT(user))
+/obj/machinery/vending/theomat/attackby(obj/item/I, mob/user)
+	if(!I.tool_qualities && check_NT(user))
```

Comment:
Can only use non-tool and only if NT disciple? Why?

Reply:
Ah good catch, those two should be separate.